lib/connection/result/result.js

```diff
@@ -113,6 +114,7 @@ function Result(options) {
       mapColumnNameToIndices[column.getName()] || [];
     mapColumnNameToIndices[column.getName()].push(index);
   }
+  Logger.getInstance().trace(`Finished mapping columns.`);
```

Comment:
It's better to use singlequotes for strings without vars.

Reply:
fixed